cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,3 +1,4 @@
+# This file should be kept in sync with cluster/images/hyperkube/dashboard-rc.yaml
```

Comment:
Line 5 also says the same about files in gce/coreos/kube-manifests/addons/dashboard. Can you merge them in one?